### PR TITLE
Adding initial support for MySQL access.

### PR DESCRIPTION
**Pull request by agorina**

### CODE REVIEW

File: exam-processor/src/main/java/org/rdw/ingest/processor/repositories/impl/AssessmentRepositoryImpl.java [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+                .addValue("name", assessment.getName())
+                .addValue("label", assessment.getLabel())
+                .addValue("version", assessment.getVersion());
+
+        jdbcTemplate.update(sql, parameterSource, keyHolder);
+        return Assessment.builder().copy(assessment).id(keyHolder.getKey().longValue()).build();
+
+    }
+
+    private static class AssessmentRowMapper implements RowMapper<Assessment> {
```
Comment (malaffoon) [inline exam-processor/src/main/java/org/rdw/ingest/processor/repositories/impl/AssessmentRepositoryImpl.java]: ugh

Comment (rcordeau) [inline exam-processor/src/main/java/org/rdw/ingest/processor/repositories/impl/AssessmentRepositoryImpl.java]: What's the matter? Don't like RowMapper?

File: exam-processor/src/test/resources/application.yml [hunk 2]
```diff
@@ -0,0 +1,9 @@
+
+spring:
+  datasource:
+    url: jdbc:mysql://localhost:3306/warehouse?useSSL=false
```
Comment (malaffoon) [inline exam-processor/src/test/resources/application.yml]: shouldn't this have a different schema or something?  if it is exactly the same as non-test, why have a separate file?

File: exam-processor/src/main/java/org/rdw/ingest/processor/repositories/impl/AssessmentRepositoryImpl.java (hunk 1 above)
Comment (agorina) [inline exam-processor/src/main/java/org/rdw/ingest/processor/repositories/impl/AssessmentRepositoryImpl.java]: sorry, I close the request. I will re-open it though. Do you have a better alternative?

Comment (malaffoon) [inline exam-processor/src/main/java/org/rdw/ingest/processor/repositories/impl/AssessmentRepositoryImpl.java]: previous life and jpa ... not sure i'm going to be happy with our decision to not use it

File: exam-processor/src/test/resources/application.yml (hunk 2 above)
Comment (agorina) [inline exam-processor/src/test/resources/application.yml]: Yes, later - just wanted to keep it simple

File: exam-processor/src/main/java/org/rdw/ingest/processor/repositories/impl/AssessmentRepositoryImpl.java (hunk 1 above)
Comment (malaffoon) [inline exam-processor/src/main/java/org/rdw/ingest/processor/repositories/impl/AssessmentRepositoryImpl.java]: no @agorina, just venting

**Pull request merged**